vector/src/main/java/im/vector/app/features/home/room/detail/composer/MessageComposerAction.kt

```diff
@@ -42,4 +42,5 @@ sealed class MessageComposerAction : VectorViewModelAction {
     data class EndAllVoiceActions(val deleteRecord: Boolean = true) : MessageComposerAction()
     data class VoiceWaveformTouchedUp(val eventId: String, val duration: Int, val percentage: Float) : MessageComposerAction()
     data class VoiceWaveformMovedTo(val eventId: String, val duration: Int, val percentage: Float) : MessageComposerAction()
+    data class AudioSeekBarMovedTo(val eventId: String, val duration: Int, val percentage: Float) : MessageComposerAction()
```

Comment:
I am wondering if this is really an action of the `MessageComposerView`? Does it handle the move of the seek bar inside the timeline or when composing a new message? The question is the same for `VoiceWaveformTouchedUp` and `VoiceWaveformMovedTo`. And maybe others but I don't have all the understanding of the feature.

Reply:
You make a strong point! It doesn't sound like this and some of the other voice actinos like the ones you mentioned and ones concerning play/pause functionality should belong here. Perhaps I could work it in another PR to separate these out a little bit